release/release_logs/1.7.0/microbenchmark.txt

```diff
@@ -0,0 +1,20 @@
+multi client tasks async per second 38216.25 +- 1653.24
```

Comment:
in 1.6.0 release logs, I saw lines for single client like `single client get calls per second 35445.18 +- 479.15
`,  but it's not here. Is this expected?

Reply:
@mwtian I remember we removed one test in the past. Can you double check if this test is needed?

Reply:
The test results here look unexpected. I think we are missing the first 10 initial entries: [1.6.0](https://github.com/ray-project/ray/blob/d202749c3254d34a71a6883b86c772b850b037d6/release/release_logs/1.6.0/microbenchmark.txt)

Reply:
OK, the reason is that we only get the last 50 lines of test output so it's truncated. I'll fix that.